src/lib.rs

```diff
@@ -89,23 +83,24 @@ fn assign_color_to_ai(str: String, human: u8) -> Goban {
     let enemy = BitBoard::from_str(&str.replace("1", "0").replace("2", "1"));
 
     if human == WHITE {
-        Goban::new(enemy, player)
-    } else {
         Goban::new(player, enemy)
+    } else {
+        Goban::new(enemy, player)
     }
 }
 
 fn launch_ai(input: Goban, player_captures: u8, opponent_captures: u8) -> (u32, u32) {
     let mut algorithm = Algorithm::new();
     algorithm.update_initial_state(input, BitBoard::empty(), player_captures, opponent_captures);
-    let ret = algorithm.get_next_move(DEPTH).unwrap();
+    let ret = algorithm.get_next_move(DEPTH, Algorithms::Minimax).unwrap();
 
     get_move_coord(&ret)
 }
 
 fn get_move_coord(node: &Node) -> (u32, u32) {
     let move_to_play = node.get_last_move();
 
+    let i : u32 = *move_to_play.get_bit_indexes().last().unwrap() as u32;
```

Comment:
How is it even possible for this to happen?

I don't want to play the cop here but @tuturr, you took the responsibility to review (and approve, it really means something serious, don't approve any PR unless you are **absolutely** sure that everything is OK), let's say it's ok for this one as you are still new at this but you should not have approved this PR, because you should have performed a real code review and saw this line duplication. Yes, it's boring, nobody wants to do a code review, but if nobody actually does I can assure you that our project is really far from being functional.

For the next time, you have to know that by approving a PR, you actually say that you are taking part of the responsibility of the said PR. In other words, if something in a PR you approved breaks the product then you are as blamable as the author of the PR.

For anyone reading this comment (cc @Ragarnoy, @abassibe), as a reminder, doing code review is a real job, not something we do waiting for a build to complete and thinking about our next meal. We have (I included of course) to be dedicated to the task and really careful because it's the very last safety we have before considering the code as done and working properly.